Server 1633 (#6462)
circleci/circleci-docs: jekyll/_cci2/server-3-install-build-services.adoc

image::realitycheck-pipeline.png[Screenshot showing the realitycheck project building in the CircleCI app]

==== Enabling Nomad Client for External Image Repository

> For enabling Nomad Client to download images from External or Private repo, You can follow below step -

Reviewer: ```suggestion
To enable the Nomad Client so it can download images from an external or private repo, follow the step below:
```

> * Make Image Registry Public to Private/Internal Network so that Nomad client can download them without any authentication.

Reviewer: ```suggestion
* Change the image registry from `Public` to `Private/Internal Network` so the Nomad client can download the registry without any authentication.
```

> * Or, Embed the Private Image repository credentials in Nomad Client Image at `/root/.docker/config.json` assuming credentials are static.

Reviewer: ```suggestion
* Or, embed the private image repository credentials in the Nomad Client Image at `/root/.docker/config.json`, assuming the credentials are static.
```

> NOTE: If you are using AWS, GCP or any other cloud image registry, most of them are using temporary docker credentials valid only for few hours to use. As soon as those credentials are expired, your nomad client will be unable to download the image from private image repository if image is not existing in Nomad client.

Reviewer: ```suggestion
NOTE: If you are using AWS, GCP or any other cloud image registry, most of these registries use temporary docker credentials valid only for few hours to use. As soon as those credentials expire, your nomad client will be unable to download the image from the private image repository if the image does not already exist in the Nomad client.
```

> NOTE: If you are using AWS, GCP or any other cloud image registry, most of them are using temporary docker credentials valid only for few hours to use. As soon as those credentials are expired, your nomad client will be unable to download the image from private image repository if image is not existing in Nomad client.

Author: @ryanwohara yeee like i tried to explain [above](https://github.com/circleci/circleci-docs/pull/6462#issuecomment-1057470846), somehow @atulsingh0's work is captured here and i can't seem to get rid of it so i'm just waiting till his PR is merged before I merge this.

> NOTE: If you are using AWS, GCP or any other cloud image registry, most of them are using temporary docker credentials valid only for few hours to use. As soon as those credentials are expired, your nomad client will be unable to download the image from private image repository if image is not existing in Nomad client.

Author: fixed!
